README.md

```diff
@@ -1,30 +1,47 @@
 # Calico VPP dataplane
-<img src="http://docs.projectcalico.org/images/felix.png" width="100" height="100">
+
+<p float="left">
+<img src="https://docs.tigera.io/img/favicon.png" width="100" height="100">
+<img src="https://fd.io/img/fdio-color.svg" width="100" height="100">
+</p>
 
 This repository contains the source for Calico's VPP dataplane integration. The integration is in incubation status, with significant development in progress.
 
+### Integration goals
+
+The main goal of this integration is to accelerate the networking of k8s clusters that use Calico thanks to [FD.io VPP](https://fd.io/docs/vpp/master/). Nodes running the VPP dataplane provide faster networking to their pods, without requiring any changes to the applications running in the pods. 
+
+This integration aims to be as transparent as possible. In particular, the VPP dataplane does not have any additional requirements compared to regular Calico. All the networking configuration, including traffic encapsulation, IP Pools, BGP Configuration, etc. is done through regular Calico means. As a result, the docs present in this repository will only describe the VPP-specific elements.
+
+VPP-enabled nodes are entirely compatible with regular Calico nodes - meaning that it is possible to have a cluster with some VPP-enabled nodes and some regular nodes. This makes it easy to migrate a cluster from Linux or eBPF Calico networking to VPP-accelerated networking.
+
+In addition to that, the VPP Dataplane provides some additional features that are not available in Calico. For instance, we currently support very fast container traffic encryption with IPsec. Network intensive applications can also require `memif` packet interfaces for optimized user-space networking.
```

Comment:
We should probably expand this section to mention multi-net, host stack and SRv6, which are arguably more important than IPsec.